packages/fxa-auth-server/test/scripts/bulk-mailer.js

```diff
@@ -179,62 +172,82 @@ describe('#integration - scripts/bulk-mailer', function () {
+          )
           .toString();
         assert.include(test1Html, 'Password changed successfully');
         const test1Text = fs
-          .readFileSync(path.join(OUTPUT_DIRECTORY, 'user1@test.com.txt'))
+          .readFileSync(
+            path.join(OUTPUT_DIRECTORY, `${account1Mock.email}.txt`)
+          )
           .toString();
         assert.include(test1Text, 'Password changed successfully');
 
         assert.isTrue(
-          fs.existsSync(path.join(OUTPUT_DIRECTORY, 'user1@test.com.headers'))
+          fs.existsSync(
+            path.join(OUTPUT_DIRECTORY, `${account1Mock.email}.headers`)
+          )
         );
         assert.isTrue(
-          fs.existsSync(path.join(OUTPUT_DIRECTORY, 'user1@test.com.html'))
+          fs.existsSync(
+            path.join(OUTPUT_DIRECTORY, `${account1Mock.email}.html`)
+          )
         );
         assert.isTrue(
-          fs.existsSync(path.join(OUTPUT_DIRECTORY, 'user1@test.com.txt'))
+          fs.existsSync(
+            path.join(OUTPUT_DIRECTORY, `${account1Mock.email}.txt`)
+          )
         );
 
         // emails are in spanish
         const test2Html = fs
-          .readFileSync(path.join(OUTPUT_DIRECTORY, 'user2@test.com.html'))
+          .readFileSync(
+            path.join(OUTPUT_DIRECTORY, `${account2Mock.email}.html`)
+          )
           .toString();
         assert.include(test2Html, 'Has cambiado la contraseña correctamente');
         const test2Text = fs
-          .readFileSync(path.join(OUTPUT_DIRECTORY, 'user2@test.com.txt'))
+          .readFileSync(
+            path.join(OUTPUT_DIRECTORY, `${account2Mock.email}.txt`)
+          )
           .toString();
         assert.include(test2Text, 'Has cambiado la contraseña correctamente');
       });
   });
 
-  it('succeeds with valid input file and method, writing emails to stdout', () => {
-    return cp
-      .execAsync(
-        `node -r esbuild-register scripts/bulk-mailer --input ${USER_DUMP_PATH} --method sendPasswordChangedEmail`,
-        execOptions
-      )
-      .then(({ stdout: result }) => {
-        assert.include(result, account1Mock.uid);
-        assert.include(result, account1Mock.email);
-        assert.include(result, 'Password changed successfully');
-
-        assert.include(result, account2Mock.uid);
-        assert.include(result, account2Mock.email);
-        assert.include(result, 'Has cambiado la contraseña correctamente');
-      });
+  it('succeeds with valid input file and method, writing emails to stdout', async () => {
+    const output = await cp.execAsync(
+      `node -r esbuild-register scripts/bulk-mailer --input ${USER_DUMP_PATH} --method sendPasswordChangedEmail`,
+      execOptions
+    );
+    const result = output.stdout.toString();
+
+    assert.include(result, account1Mock.uid);
+    assert.include(result, account1Mock.email);
+    assert.include(result, 'Password changed successfully');
+
+    // For some reason this assert fails locally
```

Comment:
Not sure what is changing this behavior, but I think its ok since we don't really need to dump files to disk.

Reply:
It appears that the output is just being truncated. Typically you'll grab output from stdout with a `stdout.data.on(cb)`. I suspect it's only getting the first chunk of data. I manually ran the command, and can validate that the full output is eventualy generated. Maybe try the `stdout.data.on` approach?